pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -47,9 +47,11 @@ type Controller struct {
 	// Arguments to use when starting new controllers
 	controllerConfig *util.ControllerConfig
 
-	// The namespace placement resource which will be needed to start
-	// sync controllers when running namespaced.
-	namespacePlacement *metav1.APIResource
+	// The federated namespace api resource will be needed to start
+	// sync controllers for namespaced federated types.  Federated
+	// namespace placement is used in determining placement of
```

Comment:
[Action not necessary] I believe this means the `placement` field in the actual API resource. Consider rephrasing.

Reply:
Updated, PTAL